src/org/usfirst/frc/team694/robot/commands/SpatulaFlipToggleCommand.java

```diff
@@ -0,0 +1,20 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+
+/**
+ *
+ */
+public class SpatulaFlipToggleCommand extends InstantCommand {
+
+    public SpatulaFlipToggleCommand() {
+        super();
+        requires(Robot.spatula);
+    }
+    
+    protected void initialize() {
+        Robot.spatula.toggle();
+    }
```

Comment:
What is the point of this? Toggling the spatula should involve the Crab Arms, this needs to add that functionality too.